scripts/config_test_vm.psm1

```diff
@@ -210,6 +211,52 @@ function Export-BuildArtifactsToVMs
     Remove-Item -Force $tempFileName
 }
 
+#
+# Install eBPF components on VM.
+#
+
+function Install-eBPFComponentsOnVM
```

Comment:
is this function changed or just moved from below?

Reply:
the diff should show changes, then moved to be contextually close to the new uninstall and better match the flow sequence